docs/ios/6x/automatic-instrumentation/hang-detection.md

```diff
@@ -0,0 +1,425 @@
+---
+title: Hang Detection
+description: Automatically detect and monitor main thread hangs in your iOS app with Embrace
+sidebar_position: 7
+---
+
+# Hang Detection
+
+The Embrace SDK automatically monitors your application's main thread for hangs (also known as Application Not Responding or ANR events), providing visibility into UI freezes and unresponsive behavior that can frustrate users.
```

Comment:
On Apple platforms, they are actually not known as ANR's, that's a term google invented. They're called App Hangs.